src/test/java/tests/boardTests/listTests/cardTests/CardTests.java

```diff
@@ -173,8 +173,33 @@ public void searchByContent() {
         searchCardResult = trelloHomePage.getTrelloHomeHeaderFragment().getListOfSearchResultTitles();
 
         softAssert.assertTrue(searchBoardResult.stream().anyMatch(genre -> genre.equals(boardName))
-                ,"No such result with name: " + boardName);
+                , "No such result with name: " + boardName);
         softAssert.assertTrue(searchCardResult.stream().anyMatch(genre -> genre.equals(cardName))
-                ,"No such result with name: " + cardName);
+                , "No such result with name: " + cardName);
+    }
+
+    @Test(description = "Delete attachment on card")
+    @Description("PJ2024-43")
+    public void deleteExistingAttachment() {
+        idCard = apiCardClient.createNewCard(cardBody, listId, HTTP_OK).getId();
+        String boardName = boardBody.getName();
+        List<String> allAttachmentTitle;
+
+        trelloHomePage.getAllBoardsFragment().specialBoardTitle(boardName).click();
+
+        AttachmentBuilder attachmentBody = AttachmentBuilder.builder().build();
+        String idAttachment = apiCardClient.createAttachmentOnCard(attachmentBody, idCard, HTTP_OK).getId();
+        AttachmentBuilder secondAttachmentBody = AttachmentBuilder.builder().build();
+        apiCardClient.createAttachmentOnCard(secondAttachmentBody, idCard, HTTP_OK).getId();
+
+        boardPage.getBoardWorkSpaceFragment().getSpecificCardTitleInList(listBody.getName(), cardBody.getName()).click();
+
+        apiCardClient.deleteExistingAttachment(idCard, idAttachment, HTTP_OK);
+        refresh();
+
+        allAttachmentTitle = boardPage.getCardFragment().getAttachmentTitles();
+
+        softAssert.assertTrue(allAttachmentTitle.stream().noneMatch(genre -> genre.equals(attachmentBody.getName())),
+                "Is attachment with name: " + attachmentBody.getName());
     }
```

Comment:
```suggestion
        softAssert.assertTrue(allAttachmentTitle.stream().noneMatch(genre -> genre.equals(attachmentBody.getName())),
                "Attachment with name: " + attachmentBody.getName() + "does'nt deleted");
    }
```
U better know, how to write it more gramarly correct